Keen.Net/ProjectSettingsProviderFile.cs

```diff
@@ -15,11 +12,14 @@ public class ProjectSettingsProviderFile : ProjectSettingsProvider
         /// <summary>
         /// <para>Reads the project settings from a text file.</para>
         /// <para>Each setting takes one line, in the order Project ID, 
-        /// Master Key, Write Key ReadKey. Unused values should be represented
+        /// Master Key, Write Key, Read Key. Unused values should be represented
         /// with a blank line.</para>
         /// </summary>
         public ProjectSettingsProviderFile(string filePath)
         {
+            // TODO : Add Keen Server URL as one of the lines, optionally.
```

Comment:
Yeah if you think they're all valid. I didn't want to create Issues if I was just thinking about it all wrong :)